tests/test_ishelper_android_arm.py

```diff
@@ -18,6 +19,7 @@
 class AndroidARMTests(unittest.TestCase):
 
     def test_check_inputstream_mpd(self):
+        delete_cached()
```

Comment:
This only needs to happen once per class either in the `setUp()` or in the `tearDown()`method as it is fine to keep it cached within the class. But this will work as well :-)

Reply:
Didn't know that, even better.

Reply:
This is very specific to **unittest** and we already do this fro VRT NU to test our code using different settings so we get coverage for bits of code hidden behind global (e.g. proxy) or add-on settings.